bot/processors/pennychat.py

```diff
@@ -435,26 +436,63 @@ def share(self, event):
                 ),
             )
             return
-        user = self.slack.users_info(user=penny_chat.user)['user']
-        for share_to in penny_chat.channels.split(',') + penny_chat.invitees.split(','):
+
```

Comment:
I added the _real_ PennyChat creation and  Participant creation from here (it is the share - that seems a better place to make this real)